RandBLAS/skge.hh

```diff
@@ -596,6 +596,84 @@ void sketch_general(
     return sketch_general(layout, opA, opS, m, d, n, alpha, A, lda, S, 0, 0, beta, B, ldb);
 };
 
+// =============================================================================
+/// \fn sketch_vector(blas::Op opS, int64_t d, int64_t m, T alpha, SKOP &S,
+///    int64_t i_off, int64_t j_off, const T *x, incx, T beta, T *y, incy
+/// )
+/// Perform a GEMV-like operation
+/// @verbatim embed:rst:leading-slashes
+/// .. math::
+///     \mat(y) = \alpha \cdot \underbrace{\op(S)}_{d \times m} \cdot \underbrace{\mat(x)}_{m \times 1} + \beta \cdot \underbrace{\mat(y)}_{d \times 1},    \tag{$\star$}
+/// @endverbatim
+/// where \math{\alpha} and \math{\beta} are real scalars and \math{S} is a sketching operator.
+/// 
+/// @verbatim embed:rst:leading-slashes
+/// What are :math:`\mat(x)` and :math:`\mat(y)`?
+///     Their shapes are defined as tall vectors of dimension :math:`(\mat(x), m \times 1)`, :math:`(\mat(y), d \times 1)`.
+///     Their precise contents are determined by :math:`(x, incx)`, :math:`(y, incy)`,
+///     and "layout = RowMajor", following the same convention as BLAS.
```

Comment:
This is a little confusing. I get that since you've declared them as column vectors, the increment is effectively a stride in a row-major representation, but this is not the usual way of describing things.

I suggest that you explicitly acknowledge this presentation as unusual. Say that we choose this representation so that ``incx`` and ``incy`` can be interpreted like ``lda`` in GEMM. Emphasize that this function treats (x, incx), (y, incy) in the exact same way as _Netlib_ BLAS (which, of course, uses column-major layout). For interfaces like CBLAS which allow either row-major or column-major layout, the layout argument in GEMV only affects the matrix (see [here](https://www.intel.com/content/www/us/en/docs/onemkl/developer-reference-c/2023-0/cblas-gemv.html) for an example). Explain that we have no ``Layout`` argument in ``sketch_vector`` because our matrix is a sketching operator, and RandBLAS never speaks of the "layout" of a sketching operator.

^ That's pretty verbose, but you get the idea.